internal/venice-common/src/main/java/com/linkedin/venice/authentication/jwt/ConfigKeys.java

```diff
@@ -0,0 +1,12 @@
+package com.linkedin.venice.authentication.jwt;
+
+public final class ConfigKeys {
```

Comment:
Nitpick: Can we call this `JwtConfigKeys` or similar? Otherwise, auto-completion will bring up many `ConfigKeys` which is a bit annoying.